src/transaction/log_replication_atomic.cpp

```diff
@@ -220,4 +244,10 @@ namespace cublog
 	  }
       }
   }
+
+  log_lsa
+  atomic_replicator::get_lowest_unapplied_lsa () const
+  {
+    return m_lowest_unapplied_lsa;
```

Comment:
Access to this variable needs to be synchronized with a mutex. Similar to `m_redo_lsa_mutex`.

Reply:
What is the response in case there was never an atomic sequence processed?
What is the response if there was an atomic sequence present long ago - in relative replication times - in the past and, in the meantime, the replication has clearly advanced enough for that value to not be relevant at all?

Reply:
The value is updated to `m_redo_lsa` if no replication is in progress, it is also initialized in the same way as `m_redo_lsa`. I will add an access mutex.